content/statement/Skycoin Launches A Vision For The New Internet.pt.md

```diff
@@ -10,24 +10,24 @@ categories = [
+A Skycoin atinge transações 100% privadas e anónimas graças ao uso do protocolo CoinJoin para combinar transações de mais de uma carteira. Ela também proporciona um ambiente de transações seguras que utiliza criptografia para se proteger de ataques de 51% e outros tipos de ameaças comuns.
 
-O ecosistema que engloba a Skycoin está se expandindo. Desenvolvedores estão criando aplicativos e ferramentas que incluem redes sociais, redes privadas (VPN) e mensageiros.
+O ecosistema que engloba a Skycoin está a expandir-se. Desenvolvedores estão criando aplicativos e ferramentas que incluem redes sociais, redes privadas (VPN) e mensageiros baseados na Skycoin.
 
-A Skycoin irá nivelar o mercado e criar um sistema de internet descentralizada distribuindo valor entre comunidades peer-to-peer. Ela entrega recursos públicos que eliminam a mineração e inflação que infestam outras criptomoedas.
+A Skycoin irá nivelar o mercado e criar um sistema de internet descentralizada distribuindo valor entre a comunidade peer-to-peer. Ela proporciona recursos públicos que eliminam a mineração e inflação que infestam outras criptomoedas.
 
-Mas os desenvolvedores dizem que o projeto Skycoin não é somente sobre criptomoeda. É também o combustível por trás da Skywire, uma rede peer-to-peer que corajosamente procura substituir a internet por algo melhor.
+Mas os desenvolvedores dizem que o projeto Skycoin não é somente sobre criptomoeda. É também o combustível por trás da Skywire, uma rede peer-to-peer que corajosamente procura substituir o cenário da internet atual por algo melhor.
 
-"Existem outros repositórios no [github.com/skycoin](https://github.com/skycoin) para redes em malha e protótipos de VPN distribuídas em que pessoas são pagas em tokens para encaminhar tráfego", diz Synth, o fundador do projeto Skycoin. "Existe também um serviço de distribuição de mídia social com replicação de dados peer-to-peer e diferentes outros projetos experimentais, assim como pesquisa em estruturas de dados imutáveis para a Internet da nova geração. Alguns deles são muito radicais."
+"Existem outros repositórios em [github.com/skycoin](https://github.com/skycoin) para redes em malha e protótipos de VPN distribuídas em que as pessoas serão pagas em tokens para encaminhar tráfego", diz Synth, o fundador do projeto Skycoin. "Existe também um serviço de média social distribuido com replicação de dados peer-to-peer e diferentes outros projetos experimentais, assim como investigação em estruturas de dados imutáveis para a Internet da nova geração. Alguns deles são muito radicais."
 
-Usuários da Skywire irão receber moedas para compartilhar conteúdo e serviços. Dessa maneira a Skycoin convida os usuários para participar de uma missão ainda maior: Remover o controle dos servidores de internet corporativos e entregar ele de volta a comunidade. É o que faz da Skycoin ser um projeto único com um propósito social.
```

Comment:
Usuários e utilizadores é basicamente a mesma coisa.
de uma missão para numa missão ao meu ver foi uma piora no contexto. Numa seria em uma. Quem participa, participa de algo ou em algo, ambos estão corretos, mas pelo menos para portugues - BR ficou estranho.
Eu mudei um pouco o sentido da ultima frase pra ficar mais correta no contexto. Sua mudança foi basicamente a tradução literal, que eu não gostei muito quando eu escrevi, por isso troquei para essa outra que pareceu mais forte.

Reply:
É o tal sentido brasileiro das frases e das palavras... em + uma = numa
usuários não se usa em portugal...
"It’s a shared social purpose that makes the Skycoin project unique."
Então fica com mais sentido:
 É o que faz da Skycoin ser um projeto único com um propósito social compartilhado.
e falta o compartilhado